README.md

```diff
@@ -214,6 +216,24 @@ async def resolve_whoami(parent, args, context, info) -> str:
 
 See also [Requests](https://www.starlette.io/requests/) in the Starlette documentation.
 
+#### Providing additional context to resolvers
+
+If you need to, you can provide your own services, functions or data to resolvers with the `context` option:
+
+```python
+from .db import database  # Imaginary
+
+@Resolver("Query.human")
+async def resolve_human(parent, args, context, info):
+    db = context["db"]
+    # Fetch human…
+
+graphql = TartifletteApp(
+  # ...,
+  context={"db": database},
+)
+```
```

Comment:
@tsunammis I'm thinking that the example I listed here could be better served via Starlette's [Other state](https://www.starlette.io/requests/#other-state) feature. Using it with `context` could lead users into thinking that `context` makes `req.state` irrelevant (when in fact it is kind of complementary).

Do you have any better example based on your experience with `tartiflette-aiohttp`? Ideally something that could also not just be imported into `resolvers.py`. I've been struggling to find such an example, to be honest.

Reply:
I think that if it can be done the "starlette" way it should be done that way. Do what you think will best fit the starlette community/API.
The idea is to have the same "features" not the same "implem" :).

Reply:
Okay, so I think it's best to leave `context` out, and document that people are encouraged to use Starlette middleware and `request.state` instead. :-)